doc/source/serve/autoscaling.md

```diff
@@ -0,0 +1,43 @@
+(serve-autoscaling)=
+
+# Serve Autoscaling
+
+This section should help you:
+
+- Understand how Ray Serve autoscaling works
+- Learn how to configure the parameters for your workload
+
+
+## Autoscaling architecture
+Ray Serve autoscaling feature automatically increases or decreases the number of replicas based on the load. 
+
+![pic](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling.svg)
+
+- Deployment handle(Client) and worker replica periodically push the stats to the autoscaler.
+- Autoscaler requires serve handle queue metrics and replicas requests metrics to make decision whether to scale (up or down) the replicas.
+- Deployment Handle(Client) keeps polling the replica stats from controller to get the updated replicas information. Serve Handle(Client) will send requests directly to the replica based on the replicas information (Round Robin).
+
+:::{note}
+When the controller dies, the client is still able to send requests, but autoscaling will pause to work. When the controller recovers, the autoscaling will start working, all previous metrics collected will be lost.
```

Comment:
In general, we can address the user directly in the docs. For instance, instead of saying "the client is still able to send requests...," we can say "you can still send requests to your deployments..." The [`ServeHandle` documentation](https://docs.ray.io/en/releases-1.13.0/serve/http-servehandle.html#servehandle-calling-deployments-from-python) is a good example for this.